pkg/kudoctl/cmd/get/get.go

```diff
@@ -74,8 +74,8 @@ func Run(args []string, opts CmdOpts) error {
 		}
 		tree.AddBranch(name)
 	}
-	_, _ = fmt.Fprintf(opts.Out, "List of current installed %s in namespace %q:\n", args[0], opts.Namespace)
-	_, _ = fmt.Fprintln(opts.Out, tree.String())
+	fmt.Fprintf(opts.Out, "List of current installed %s in namespace %q:\n", args[0], opts.Namespace)
+	fmt.Fprintln(opts.Out, tree.String())
```

Comment:
This is marked as a warning in my Goland/IntelliJ - Looks like we don't have that as a linting rule. I'm ok with either, tbh.